subcommands/targets/deltas.go

```diff
@@ -21,14 +21,14 @@ var (
 func init() {
 	var deltas = &cobra.Command{
 		Use:   "static-deltas <target-version> [<from-version>...]",
-		Short: "Generate static deltas to the given target version to make OTAs faster",
+		Short: "Generate static deltas to the given Target version to make OTAs faster",
 		Run:   doDeltas,
 		Args:  cobra.MinimumNArgs(1),
-		Long: `In many cases OTA updates will have many OSTree changes. These updates
-can be downloaded significantly faster by generating OSTree static
-deltas. Static deltas are generated with a "from(sha) -> to(sha)" type
-logic. This command takes the given Target version and will produce a
-number of static deltas to ensure devices will be updated efficiently.`,
+		Long: `In many cases OTA updates will have multiple OSTree changes. These updates
+can be downloaded faster by generating OSTree static
```

Comment:
```suggestion
can be downloaded significantly faster by generating OSTree static
```
Lets stick with significantly.